### ISSUE TITLE
.jar file fails to launch

### ISSUE THREAD

**Issue by tylerheald** [wilsontheory/binance_arbitrage#1]
I've tried both windows and Linux. Neither seems to work.
In windows I've tried the command prompt and simply running the .jar executable.
My Java runtime environment is up to date.

Suggestions?

**Comment by wilsontheory** [comment]
What kind of error is being thrown? Does the program run with a blank output?

**Comment by tylerheald** [comment]
Yes exactly. I can see Java begin to run in task manager so I figured maybe
it was just incredibly slow. Left in executed over night. No output.

**Comment by wilsontheory** [comment]
weird, so the program is probably working but the socket connection isn't being made. I don't think I can fix that on my end
